FreeRTOS/FreeRTOS
# IN_PROGRESS: vTaskGetInfo,vTaskGenericNotify,xTaskResumeFromISR,vTaskResume

Coverage testing for tasks.c

Description
-----------
Coverage tests for vTaskGetInfo,vTaskGenericNotify,xTaskResumeFromISR,vTaskResume

Test Steps
-----------
cd FreeRTOS/Test/CMock/smp
make
make lcovhtml

Related Issue
-----------
TS-24220
TS-24219
TS-24208
TS-24182

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
